src/config.rs

```diff
@@ -27,13 +27,29 @@ pub struct RpcMethod {
     #[serde(default)]
     pub params: Vec<MethodParam>,
 
+    /// if cache_ttl_seconds is defined, cache default will be 1
```

Comment:
why?

Reply:
If you're defining a ttl then you want caching, so even if you don't set the cache size it will default to 1